test-data/unit/check-kwargs.test

```diff
@@ -484,13 +484,28 @@ def f(*vargs: int, **kwargs: object) -> None:
 def g(arg: int = 0, **kwargs: object) -> None:
 	pass
 
+def h(*, arg: int = 0, **kwargs: object) -> None:
+  pass
+
+def j(arg: int, **kwargs: object) -> None:
+  pass
+
+def k(*, arg: int, **kwargs: object) -> None:
+  pass
+
 d = {} # type: Dict[str, object]
 f(**d)
-g(**d)  # E: Argument 1 to "g" has incompatible type "**Dict[str, object]"; expected "int"
+g(**d)
+h(**d)
+j(**d)  # E: Argument 1 to "j" has incompatible type "**Dict[str, object]"; expected "int"
```

Comment:
When functions do not have a default value - we still raise a warning. Because we are not sure that `**kwargs` will match existing argument. But, when there's a default - we have nothing to worry about in any case.